homeassistant/components/zwave_js/services.py

```diff
@@ -124,7 +124,7 @@ def async_register(self) -> None:
             const.DOMAIN,
             const.SERVICE_SET_VALUE,
             self.async_set_value,
-            schema=vol.Schema(
+            schema=vol.All(
```

Comment:
We should wrap everything in a `vol.Schema`. So please keep that and add `vol.All` in the inner level.